src/views/forms/TriggersForm/TriggerOrderInputs.tsx

```diff
@@ -149,58 +195,107 @@ export const TriggerOrderInputs = ({
     </Styled.MultipleOrdersContainer>
   );
 
-  return (
+  const headerTooltip = () => (
+    <WithTooltip tooltip={tooltipId}>{stringGetter({ key: stringKeys.header })}</WithTooltip>
+  );
+
+  return isMultiple ? (
+    <Styled.TriggerRow key={tooltipId}>
+      <Styled.Heading>{headerTooltip()}</Styled.Heading>
+      <Styled.InlineRow>{multipleOrdersButton()}</Styled.InlineRow>
+    </Styled.TriggerRow>
+  ) : (
     <Styled.TriggerRow key={tooltipId}>
-      <WithTooltip tooltip={tooltipId}>
-        <h3>{stringGetter({ key: stringKeys.header })}</h3>
-      </WithTooltip>
+      <Styled.Heading>
+        {headerTooltip()}
+        <Styled.HeadingInfo>
+          {stringGetter({ key: stringKeys.headerDiff })}
+          {signedOutput()}
+          <Styled.VerticalSeparator />
+          <Styled.ClearButton
+            action={ButtonAction.Destroy}
+            size={ButtonSize.Base}
+            type={ButtonType.Button}
+            onClick={clearPriceInputFields}
+          >
+            {stringGetter({ key: STRING_KEYS.CLEAR })}
+          </Styled.ClearButton>
```

Comment:
should we conditionally show this based on whether the inputs are filled?

Reply:
hmm I feel like it might look a lil visually jumpy - lemme follow up cuz I gotta make a separate PR for a new abacus bump